packages/lodestar/src/network/encoders/utils.ts

```diff
@@ -6,14 +6,14 @@ import {SnappyFramesUncompress} from "./snappy-frames/uncompress";
 
 export function getCompressor(encoding: ReqRespEncoding): (data: Buffer) => AsyncGenerator<Buffer> {
   switch (encoding) {
-    case ReqRespEncoding.SSZ_SNAPPY: {
```

Comment:
We don't have rule to enforce brackets on case statements. :disappointed: 
Brackets really helps out to notice case boundaries.

Reply:
For switch statements with a lot of short cases not using brackets is pretty clean. If the case code grows brackets are nice so I'm not sure about making them mandatory